lib/pbench/server/api/resources/endpoint_configure.py

```diff
@@ -0,0 +1,106 @@
+from flask_restful import Resource, abort
+from flask import jsonify
+
+from pbench.server.api.resources.query_apis import get_index_prefix
+
+
+class EndpointConfig(Resource):
+    """
+    EndpointConfig API resource: this supports dynamic dashboard configuration
+    from the Pbench server rather than constructing a disconnected dashboard
+    config file.
+    """
+
+    def __init__(self, config, logger):
+        """
+        __init__ Construct the API resource
+
+        Args:
+            config (PbenchServerConfig): server config values
+            logger (Logger): message logging
+        """
+        self.logger = logger
+        host = config.get("pbench-server", "proxy_host")
```

Comment:
This means it is always required to have this in the configuration file.  Could we make this optional?

Reply:
I put it in the default file so I'd have a place to explain it; that'd be harder to do if it was omitted entirely, and it didn't seem to have any real downside. Do you object?